introduction.md

```diff
@@ -1,69 +1,66 @@
-
-The content is split into four parts: the first is dedicated to familiarizing yoursefl with C++ jargon, syntax, and idioms.
-The second part will go over structure multi-shource projects; well organized code helps both the development process and project comprehension by others.
-We will also discuss documentation of code in this section.
-The third  part is dedicated to giving some example projects and to discuss design considerations.
-Lastly, the fourth  part digs into remote computing, bash scripting and slurm.
-
-### C++ lessons
-
-1. What is a program
-1. Compilation and debugging/debuggers
+- Project Configuration
+- Version Control with Git, including remote server visualization with GitHub
+
+The standard for high performing scientific computing is C++.
+Therefore, the primary focus of the following sections will be to provide guidance for the C++ programming language. 
+Examples in this document will serve to as a template for configuring and setting up more complicated projects. 
+Let it be noted that the following content is subjective and not intended to be comprehensive.
+It reflects personal preferences and conclusions regarding the C++ language.
+That is, C++ is a a very large and general purpose language, allowing for freedom of expression and emphasis. 
+To this extent, little importance is placed on concepts such as inheritance, highlighting its relevance simply for applications such as "Curiously Recurring Template Parameters". 
+Furthermore, this document will address modern C++, encompassing language constructs from 2011 and onward, and a review of the standard library.
+
+## Chapters and Content
+
+The content will be divided into 4 parts:
+- C++ jargon, syntax, an idioms
+- Structure and documentation of multi source projects
+- Example projects and design consideration
+- Command Line, Remote Computing, bash scripting, Slurm, and Git
+
+### C++
+
+1. What is a program?
+1. Compilation and debugging
 1. Variables, types, values and templates
-1. Functions, operators, lambdas (anonymous functions), function overloading, and variadic templates
+1. Functions, operators, lambdas, function overloading, and variadic templates
 1. User-defined types (classes)
 1. Pointers, arrays, references, and smart-pointers
-1. Memory management and memory owndership models
-1. Digression: imperative programming vs object-oriented programming vs functional programming
+1. Memory management and owndership models
+1. Digression/Paradigms: Imperative vs Object-Oriented vs Functional Programming
 1. Move/copy construction/assignment
-1. Our best friend, std::optional, and error handling
-1. More on the standard template library (STL
+1. std::optional and error handling
+1. More on the standard template library (STL)
 
-What we will not cover is:
+ Topics briefly considered are:
```

Comment:
No, I really don't plan to talk about these.
Maybe in future book-blog posts, but not this one.